Expensify/App
# Fix link to oldDot

### Details


### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/177739

### Tests

I tested both scenarios below on dev with carlos@expensifail.com and carlos+test@expensifail.com after @marcaaron pointed out that it wasn't working for him with an email that had a plus sign in it. So I added the encodeURIComponent call and tested with carlos+test. 

**That said, this only works if the account clicking manage cards is a domain admin, so this isn't quite working for all cases**

- Had to edit some code to see the option to create workspace (but I think that's fine)
- Sign out of e.com
- Click on big + button
- Click Create workspace
- Click manage cards
- Check it opened the correct page on e.com and that you are signed in
- I also live edited the code to link to the settings page (since that is used in some other flow)

---

- Clear all cookies on e.com
- Click on big + button
- Click Create workspace
- Click manage cards
- Check it opened the correct page on e.com and that you are signed in


### QA Steps
Run both scenarios below with an email that doesn't have a plus sign in it, and one that does

- Sign out of e.com
- Click on big + button
- Click Create workspace
- Click manage cards
- Check it opened the correct page on e.com and that you are signed in

---
- Clear all the cookies on e.com
- Click on big + button
- Click Create workspace
- Click manage cards
- Check it opened the correct page on e.com and that you are signed in


### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
